expressionGenerator.js

```diff
@@ -0,0 +1,51 @@
+const fieldsToUpdate = (body) => {
+    var fields = ['name', 'description', 'showDate']
```

Comment:
Lendo seu código na linha 1 me faz pensar que o código é pra ser dinâmico, mas se os fields aqui são fixos, eles não fazem sentido. Você tem duas opções:

1.  Nessa versão segue a mesma lógica que usou, por que vai ser indiferente os keys do body você vai montar uma query pra atualizar
```suggestion
const fieldsToUpdate = (body, fields) => {
```

2.  Nessa versão é mais fácil por que ela ja pode fazer o spread dos itens e verificar eles um por um. Sem sofrer com for depois
```suggestion
const fieldsToUpdate = ({ name, description, showDate }) => {
    var fields = ['name', 'description', 'showDate']
```